### ISSUE TITLE
The commandlet does not exist onprem

### ISSUE THREAD

**Issue by DimaRazb** [MicrosoftDocs/office-docs-powershell#3162]
The commandlet does not exist onprem. Then, applies to line 
Applies to:
Exchange Server 2016, Exchange Server 2019, Exchange Online
should  be aligned to only EXO.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3665db68-7dd8-2e93-1797-505eb77a22ce
* Version Independent ID: 9cf5b8f2-a6aa-e79b-70e7-cfe5c1aee69c
* Content: [Get-RecoverableItems (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/mailboxes/get-recoverableitems?view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/mailboxes/Get-RecoverableItems.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/mailboxes/Get-RecoverableItems.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by yogkumgit** [comment]
@officedocsbot assign @yogkumgit

**Comment by vampire337** [comment]
Just ran into this as well.  Get-RecoverableItems does not exist on our Exchange 2016 (Cumulative Update 8, version 15.1.1415.2)

**Comment by chrisda** [comment]
@DimaRazb, @vampire337, it was available to me on Exchange 2016 CU12 and Exchange 2019 CU1 test servers. Can someone else confirm or deny?

**Comment by dariomws306** [comment]
Hi @chrisda , @DimaRazb and @vampire337 , I have ran these commands on Exchange 2016 CU10:
Get-ManagementRole -Cmdlet Restore-RecoverableItems
Get-ManagementRole -Cmdlet Get-RecoverableItems

And I can confirm that they exist on Exchange 2016.

@DimaRazb and @vampire337 Do you have the Mailbox Import Export role assigned? Could you run the commands above?

@chrisda , maybe we should add a note to this article to let them know that you need this role.

Thank you all!

**Comment by chrisda** [comment]
Yes. That appears to be the problem (cmdlet is only available via Mailbox Import Export role, which isn't assigned to any role groups by default). See #3161 for more details.

Closing

**Comment by dariomws306** [comment]
Thank you @chrisda!!

**Comment by vampire337** [comment]
@dariomws306
Yes, I have Mailbox Import Export Role assigned.  I have imported and exported a handful of PST files in the past several weeks, and I vaguely recall having to assign that role when I started the process at the beginning of March.
Yes, I am able to run those commands, but there was no response.  Should I expect a response if the role is correctly assigned?
No, Get-RecoverableItems is still not available (in CU8).  I will be upgrading to CU10 and will report back if that gives me the command.
